listado-de-libros/scss/detail.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import 'equalizer.scss';
+@import 'colors.scss';
+
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
+.nav-bar {
+  height: 70px;
+  width: 1169px;
```

Comment:
If you set this fixed size, the navbar will not properly adapt itself to other display sizes. What you can do is set it to:

`width: 100vw;` or `width: 100%;`

With the first one, you are setting it to be the width of the ViewWidth of the browser window. The second is telling the navbar to occupy the available space it has according to its parent